chia/_tests/core/mempool/test_mempool_manager.py

```diff
@@ -427,18 +436,18 @@ def make_bundle_spends_map_and_fee(
         eligibility_and_additions[coin_id] = EligibilityAndAdditions(
             is_eligible_for_dedup=bool(spend.flags & ELIGIBLE_FOR_DEDUP),
             spend_additions=spend_additions,
-            is_eligible_for_ff=bool(spend.flags & ELIGIBLE_FOR_FF),
+            ff_puzzle_hash=bytes32(spend.puzzle_hash) if bool(spend.flags & ELIGIBLE_FOR_FF) else None,
```

Comment:
what type does `spend.puzzle_hash` have? is it not `bytes32`?

Reply:
It's `bytes`: https://github.com/Chia-Network/chia_rs/blob/38d4f845705b1fd9cc68077a9d50a29afd79e6d5/wheel/python/chia_rs/chia_rs.pyi#L269

Reply:
I see. that's a problem we should address separately